dictionaries/software-terms/src/coding-compound-terms.txt

```diff
@@ -46,7 +46,7 @@
 *Files*
 *Fix*
 *Gateway*
-goog+
+Goog+
```

Comment:
You should add a comment here

Does Goog state for Google and it's variation? What is supposed to catch?